common/source/docs/common-gps-septentrio.rst

```diff
@@ -14,6 +14,31 @@ The Septentrio AsteRx-mUAS RTK GPS is a relatively expensive but also highly acc
      This GPS is supported in Copter-3.4 (and higher), Plane 3.7.0 (and higher) and Rover 3.0.1 (and higher).
      Mission Planner GCS is the only GCS to support this feature currently.
 
+Setup instructions
+==================================
+
+To setup this using GPS1 input (serial3) configure these parameters:
+
+- GPS_INJECT_TO,0
+- GPS_RATE_MS,100
+- GPS_TYPE,10
+- SERIAL3_BAUD,115
+- SERIAL3_PROTOCOL,5
+
+To setup this using GPS2 input (serial4) configure these parameters:
+
+- GPS_INJECT_TO,1
+- GPS_RATE_MS2,100
+- GPS_TYPE2,10
+- SERIAL4_BAUD,115
+- SERIAL4_PROTOCOL,5
+
+Warning
+==================================
+
+When using AsteRX-m2 with FW version 4.2.2 or newer, do not connect Mission Planner or any other ground station to ArduPilot during the first 15 seconds after booting otherwise the GPS will not get configured properly and will therefore not work.
```

Comment:
This does not match any experience I have with the GPS at all. I routinely reboot the aircraft with the GCS connected over telemetry, and not once has this resulted in a problem. Unless you have the GCS instantly injecting data this does not make sense as an explanation of what you were seeing.